torch/_inductor/freezing.py

```diff
@@ -222,7 +222,12 @@ def freeze(
     aot_autograd_gm.recompile()
 
     from torch._inductor.compile_fx import fake_tensor_prop
+    from torch._inductor.fx_passes.post_grad import view_to_reshape
 
+    # We have convert conv's weight to channels last which may meet error for .view
+    # when doing fake_tensor_prop. So we need to convert view to reshape first.
+    # see https://github.com/pytorch/pytorch/blob/main/torch/_inductor/compile_fx.py#L472.
```

Comment:
Seems not good to place a code link without commit id here (things might change in the future and the code lines don't match). How about add a "note anchor" (like `[NOTE: view to reshape conversion]`) in compile_fx.py#472 and refer to the note anchor here?

Reply:
updated.